source/agora/common/Block.d

```diff
@@ -201,17 +203,11 @@ public struct Block
         // transactions are ordered lexicographically by hash in the Merkle tree
         this.merkle_tree.sort!("a < b");
 
-        size_t j = 0;
-        for (size_t length = this.txs.length; length > 1; length = (length + 1) / 2)
+        for (size_t order = bsr(this.txs.length); order != 0; order--)
```

Comment:
I can't find the symbol `bsr`? Where is it?

Reply:
Core.bitop